app/uk/gov/hmrc/excisemovementcontrolsystemapi/controllers/DraftExciseMovementController.scala

```diff
@@ -53,8 +55,28 @@ class DraftExciseMovementController @Inject()(
 
   private def handleSuccess(implicit request: ValidatedXmlRequest[NodeSeq]): Future[Result] = {
 
+    val newMovement: Movement = createMovementFomMessage(request)
+    val saveMovementFn = movementMessageService.saveMovementMessage(newMovement)
+    val submitToNrsFn = submitToNrsService(newMovement)
+
+    //todo: Here we do not do anything id the NRS fails. THe NRS confluence page says we can do that
+    // handle the error if we want. I think the common approach by all the service is not doing anything.
+    // Also is this the right place to submit to the NRS?.
```

Comment:
The right place might be the Submit Connector? Just thinking that the messages we get in from the SubmitController (IE818 etc) will also need to be sent to NRS?